opossum-tool/OpossumUI
# feat: virtualize autocomplete

### Summary of changes

- rebuild autocomplete for virtualization with react-virtuoso using MUI useAutocomplete
- re-declare forwardRef such that it can be used together with generics

### Context and reason for change

#2385 

### How can the changes be tested

Use a large input file to notice that the autocompletes (such as "package name" and "licenses") open quickly compared to base branch.
Also test that the locator popup autocomplete still looks and works as expected.
